stubs/aiofiles/aiofiles/tempfile/temptypes.pyi

```diff
@@ -17,6 +17,7 @@ _T = TypeVar("_T")
 class AsyncSpooledTemporaryFile(AsyncBase[_T]):
     def fileno(self) -> Generator[Incomplete, Incomplete, Incomplete]: ...
     def rollover(self) -> Generator[Incomplete, Incomplete, Incomplete]: ...
+    async def close(self, *args: Incomplete, **kwargs: Incomplete) -> None: ...
```

Comment:
```suggestion
    async def close(self) -> None: ...
```
This gets delegated to the underlying file's close() method.

Reply:
Thanks, I wasn't sure of the exact signature. I'll add an allowlist entry.